client/my-sites/plans-features-main/test/index.jsx

```diff
@@ -276,4 +280,68 @@ describe( 'PlansFeaturesMain.getPlansForPlanFeatures() with tabs', () => {
 			comp.find( 'SegmentedControlItem[path="?customerType=personal"]' ).props().selected
 		).toBe( false );
 	} );
+
+	test( 'Should highlight proper popular plan for blog site type', () => {
+		const instance = new PlansFeaturesMain( {
+			siteType: 'blog',
+		} );
+		const comp = shallow( instance.render() );
+		expect( comp.find( 'PlanFeatures' ).props().popularPlanSpec ).toEqual( {
+			type: TYPE_PERSONAL,
+			group: GROUP_WPCOM,
+		} );
+	} );
+
+	test( 'Should highlight proper popular plan for business site type', () => {
```

Comment:
I think this should read `for professional site type`

Reply:
Good eye, @creativecoder -- fixed the transposed test descriptions